src/screens/ChoosePinScreen.js

```diff
@@ -46,7 +47,7 @@ class ChoosePinScreen extends React.Component {
 
   constructor(props) {
     super(props);
-    this.words = this.props.navigation.getParam('words');
+    this.words = this.props.route.params.words;
```

Comment:
Maybe use lodash `get` to prevent it from crash if it doesn't have `params`

Reply:
Added a protection with the optional chaining operator `?`